.github/workflows/ci.yml

```diff
@@ -73,6 +71,10 @@ jobs:
           weblog: spring-boot-openliberty
         - library: java
           weblog: spring-boot-undertow
+        - library: java
+          weblog: spring-boot-native
```

Comment:
Could you keep the `spring-boot` name? it is used in AppSe feature matrix

Reply:
This name defines the docker image to build the weblog variant.
How can we keep the same name?

Reply:
It's the same case as "spring-boot-undertow" and "spring-boot-openliberty", they are build profiles defined in spring-boot app. I think we can not  keep the spring-boot name....